subhub/api/payments.py

```diff
@@ -33,19 +27,14 @@ def subscribe_to_plan(uid, data) -> FlaskResponse:
             source_token=data["pmt_token"],
             origin_system=data["orig_system"],
         )
-    except InvalidRequestError as e:
-        return {"message": f"Unable to subscribe. {e}"}, 400
-    existing_plan = has_existing_plan(customer, plan_id=data["plan_id"])
-    if existing_plan:
-        return {"message": "User already subscribed."}, 409
-    if existing_plan:
-        return {"message": "User already subscribed."}, 409
-    try:
+        existing_plan = has_existing_plan(customer, plan_id=data["plan_id"])
+        if existing_plan:
+            return {"message": "User already subscribed."}, 409
         stripe.Subscription.create(
             customer=customer.id, items=[{"plan": data["plan_id"]}]
         )
-    except InvalidRequestError as e:
-        return {"message": f"Unable to subscribe: {e}"}, 400
+    except StripeError as e:
```

Comment:
Why is this catching [every single Stripe error ](https://github.com/stripe/stripe-python/blob/master/stripe/error.py)(including auth and rate limit issues in SubHub) and returning them as client errors?

Reply:
@bbangert I updated it to catch all Stripe errors, since the request was to show the Stripe error messages. I left it as a 400 error, since that was how we were handling it before. I do not mind setting it back to only catching the InvalidRequestError.